core/src/test/java/ar/com/wolox/wolmo/core/util/ImageProviderTest.kt

```diff
@@ -0,0 +1,186 @@
+package ar.com.wolox.wolmo.core.util
+
+import org.assertj.core.api.Assertions.assertThat
+import android.app.Application
+import android.content.ComponentName
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.graphics.Bitmap
+import android.net.Uri
+import android.provider.MediaStore
+import androidx.fragment.app.Fragment
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.*
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.Implementation
+import org.robolectric.annotation.Implements
+import org.robolectric.annotation.RealObject
+import org.robolectric.shadows.ShadowBitmap
+import java.io.File
+import java.io.IOException
+import java.io.OutputStream
+import java.util.HashMap
+
+@RunWith(RobolectricTestRunner::class)
+@Config(manifest = Config.NONE, shadows = [ImageProviderTest.Companion.WolmoShadowIntent::class, ImageProviderTest.Companion.WolmoShadowBitmap::class])
+class ImageProviderTest {
+
+    companion object {
+        internal var sComponentNameMock: ComponentName? = null
```

Comment:
The naming convention of these variables is off.

Reply:
ok :+1: